### DISCUSSION TITLE
Multi-tenancy and multitenant <Link to .. />

### DISCUSSION THREAD

**Discussion by sharadshahihw** [remix-run/react-router#10868] [Q&A]
Hi,

We are trying to build a multi-tenancy app. We decided not to use the `basename` and go with route with parameters.

```js

return createBrowserRouter([
        {
            children: [
                {
                    element: <SomePage/>,
                    index: true,                  
                },                
            path: '/:tenantId',
        },
    ]);

```

The issue with this is we will have to interpolate tenantId in `to` prop of all `Link` components.
```js
const {tenantId} = useParams();
<Link to=`${tenantId}/otherpage`>Other Page</Link>
```

Other way would be to make utility component that wraps it.

Are there any other ways to modify `Link to` so that we don't repeat the interpolation.

**Comment by brophdawg11** [comment]
One option would be to use [relative links](https://reactrouter.com/en/main/start/overview#relative-links) to inherit the current tenant: `<Link to="otherpage">`
